CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main / unreleased
 
+* [FEATURE] Ingester: Add optional CPU/memory utilization based read limiting, considered experimental. Disabled by default, enable via `-ingester.utilization-based-limiting-enabled` flag. #5012
```

Comment:
[nit] Since this is going to be experimental, please add it to the list of experimental features at `docs/sources/mimir/configure/about-versioning.md`.

Reply:
Done.